crates/pagecache/src/ds/lru.rs

```diff
@@ -1,16 +1,15 @@
 use parking_lot::Mutex;
 use std::ptr;
+use super::dll::{LinkedList, Node, Item};
 
-use super::*;
-
-/// A simple Lru cache.
-pub struct Lru {
+/// A simple LRU cache.
+pub struct LRU {
     shards: Vec<Mutex<Shard>>,
 }
 
-unsafe impl Sync for Lru {}
+unsafe impl Sync for LRU {}
```

Comment:
This should remain `Lru` according to https://rust-lang-nursery.github.io/api-guidelines/naming.html

Did clippy complain about it not being LRU?

Reply:
No, i just thought it’s more readable. Will turn it back.

Reply:
Personally, I agree with you and for a little bit this bothered me, but over time I became more convinced it was better to be idiomatic. I now accept my idiomatic overlords :P